_includes/footer-navigation.html

```diff
@@ -0,0 +1,21 @@
+{% assign logo = include.logo | default: 'logo.svg' %}
```

Comment:
[suggestion] it looks like we aren't passing anything in the one place we're rendering this partial. we should probably just directly set these values until we need to dynamically replace them

Reply:
yes, exactly! and yeah, we can remove `nav` entirely